examples/tree/plot_unveil_tree_structure.py

```diff
@@ -122,24 +122,35 @@
                 feature=feature[i],
                 threshold=threshold[i],
                 right=children_right[i],
-                value=values[i],
+                value=np.around(values[i], 3),
             )
         )
 
 # %%
 # What is the values array used here?
 # -----------------------------------
 # The `tree_.value` array is a 3D array of shape
-# [``n_nodes``, ``n_classes``, ``n_outputs``] which provides the count of samples
-# reaching a node for each class and for each output. Each node has a ``value``
-# array which is the number of weighted samples reaching this
-# node for each output and class.
+# [``n_nodes``, ``n_classes``, ``n_outputs``] which provides the proportion of samples
+# reaching a node for each class and for each output.
+# Each node has a ``value`` array which is the proportion of weighted samples reaching
+# this node for each output and class with respect to the parent node.
+#
+# One could convert this to the absolute weighted number of samples reaching a node,
+# by multiplying this number by `tree_.weighted_n_node_samples[node_idx]` for the
+# given node, or by `tree_.n_node_samples[node_idx]` for the unweighted number of
+# samples.
 #
 # For example, in the above tree built on the iris dataset, the root node has
-# ``value = [37, 34, 41]``, indicating there are 37 samples
+# ``value = [0.331, 0.304, 0.366]`` indicating there are 33.1% of class 0 samples,
```

Comment:
The printed output and `plot_tree` says  0.33 rather than 0.331 in the example doc.

Othere places need to be fixed similarly.

Reply:
Fixed this to say 0.33 now